public/components/visualization/text2viz.tsx

```diff
@@ -410,6 +444,13 @@ export const Text2Viz = () => {
                   paddingSize="none"
                   scrollable={false}
                 >
+                  {usageCollection ? (
```

Comment:
IIUC, usageCollection is always available, the feature flag only controls whether it can send metrics to server.

Reply:
`usageCollection` is declared as an optional plugin of `dashboards-assistant` plugin. Theoretically,  it can be undefined.